src/app/layout.tsx

```diff
@@ -1,45 +1,45 @@
-import { Inter, Lexend } from 'next/font/google'
-import clsx from 'clsx'
+import { Inter, Lexend } from "next/font/google";
+import clsx from "clsx";
```

Comment:
Git history will show that you have authored these changes, when they're only formatting changes. We want formatting changes to be invisible in the history.  Here's an article about it:  <https://github.blog/changelog/2022-03-24-ignore-commits-in-the-blame-view-beta/>  


I will go over how to fix this with you in a 1:1, so don't worry about doing this on your own.  Messing with git history can get very tricky. Wil will reset the history back to before the formatting was run and add those formatted files to their own commit with a "Format" message.  Then we can add that commit hash to a `.git-blame-ignore-revs` file.